scripts/continuous/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+# Since travis_wait doesn't work with docker well somewhy,
+# just use bell which prints something to stdout every 5 minutes.
+function bell() {
+	for i in {5..20..5}; do
```

Comment:
Why not forever?

Reply:
Hm, yes, since we assume build could not hand, infinite loop seems to be ok. Fixed

Reply:
btw, Thank you for review!